cpp/arcticdb/storage/test/test_storage_exceptions.cpp

```diff
@@ -202,8 +202,8 @@ class LMDBStorageTestBase : public ::testing::Test {
 };
 
 TEST_F(LMDBStorageTestBase, WriteMapFullError) {
-    // Create a Storage with 32KB map size
-    LMDBStorageFactory factory(32ULL * (1ULL << 10), false);
+    // Create a Storage with 64KB map size
+    LMDBStorageFactory factory(64ULL * (1ULL << 10), false);
```

Comment:
In the M1 arm environment, the 32KB map throws a map-full exception on allocation. Looks like the minimum is 64KB.